doc/ref_cert/RC2/chapters/chapter01.rst

```diff
@@ -0,0 +1,199 @@
+Introduction
+============
+
+.. figure:: ../figures/bogo_lsf.png
```

Comment:
Didn't we agree on removing the bogometers?

Reply:
yes. But github doesn't help managing PR dependencies - I might fail in the // PR here.